src/robusta/core/playbooks/playbooks_event_handler_impl.py

```diff
@@ -218,6 +220,8 @@ def __handle_findings(self, execution_event: ExecutionBaseEvent):
                     sink.write_finding(
                         finding_copy, self.registry.get_sinks().platform_enabled
                     )
+
+                    sinks_findings_dict[sink_name] += 1
```

Comment:
Would be nice the capture the sink type as well, because sink_name might hide what it is.
For example, if someone names the sink 'prod' or 'support', we won't know the sink type.
WDYT?